src/node_options.cc

```diff
@@ -368,6 +368,11 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "silence all process warnings",
             &EnvironmentOptions::no_warnings,
             kAllowedInEnvironment);
+  AddOption("--no-node-snapshot",
+            "Launch Node.js without deserializing from the Node.js custom "
```

Comment:
Might want to mention it's an undocumented debug flag here.

Reply:
I think the warnings are alphabetical so this should go above `--no-warnings`?

Reply:
Er, well, they aren't alphabetical, but they're *mostly* alphabetical?

Reply:
… if this is supposed to be undocumented, I’d prefer not documenting it at all (i.e. no help text) over documenting that it’s undocumented, because the latter is pretty self-contradictory.